Rakefile

```diff
@@ -53,6 +53,7 @@ task :generate_policy_list do
       # 'deprecated' defaults to false unless explicitly set to true
       publish = !(publish == 'false' || publish == false)
       deprecated = deprecated == 'true' || deprecated == true
+      hide_skip_approvals = pp.parsed_info[:hide_skip_approvals]
```

Comment:
Can we confirm that this will work as expected for policy templates that do not have a hide_skip_approvals in their info() block?

Reply:
Good point! Thanks for bringing it up.
I ran the Rakefile locally, and for policy templates without `hide_skip_approvals` in the info() block, the script adds it as nil in the JSON output. This aligns with how other optional fields such as `recommendation_type` are handled, so it should work as expected.
![Screenshot 2024-11-08 104542](https://github.com/user-attachments/assets/38ba325d-96d6-40a5-acca-d939c2c37341)

Reply:
For fields that toggle a setting, we want this to reflect what the policy is going to do. In those cases, it should show whatever the default functionality is if the value is not present; in this case, it should be false, similar to what you see with deprecated.

null is only used for non-boolean values that are non-applicable for some policy templates.

Reply:
got it, thanks. 
Applied at https://github.com/flexera-public/policy_templates/pull/2813/commits/755c47331075ce29b806cff633e2a1b89696c19d